docs/UI/SetMediaClockTimer/index.md

```diff
@@ -43,27 +43,17 @@ The UI.SetMediaClock timer request indicates either an initial value for the med
 
 |Name|Type|Mandatory|Description|
 |:---|:---|:--------|:---------|
-|startTime|[Common.TimeFormat]|false|_startTime_ must be provided together with modes: "COUNTUP", "COUNTDOWN", "PAUSE" to HMI. _startTime_ will be ignored for "RESUME", and "CLEAR".|
-|endTime|[Common.TimeFormat]|false|endTime can be provided together with modes: "COUNTUP", "COUNTDOWN", "PAUSE" to HMI. To be used to calculate any visual progress bar (if not provided, this feature is ignored). If endTime is greater then startTime for COUNTDOWN or less than startTime for COUNTUP, then the request will return an INVALID_DATA. _endTime_ will be ignored for "PAUSE", "RESUME", and "CLEAR".|
-|updateMode|[Common.ClockUpdateMode]|true|Enumeration to control the media clock. In case of pause, resume, or clear, the start time value is ignored and shall be left out. For resume, the time continues with the same value as it was when paused.|
-|audioStreamingIndicator|[Common.AudioStreamingIndicator]|false|Indicates that a button press of the Play/Pause button would play, pause or stop the current playback.|
+|startTime|[Common.TimeFormat](../../common/structs/#timeformat)|false|_startTime_ must be provided together with modes: "COUNTUP", "COUNTDOWN", "PAUSE" to HMI. _startTime_ will be ignored for "RESUME", and "CLEAR".|
```

Comment:
This is the only documentation I've seen containing _italics_ so I want to confirm, should we leave that in?

Reply:
Haven't seen a lot of RPC params reference themselves in their descriptions. However it does make the param names stand out so it might be a good idea to keep them in